TO-DO-APP/app/src/main/java/com/example/to_do_app/Adapter.kt

```diff
@@ -1,44 +0,0 @@
-package com.example.to_do_app
-
-import android.content.Intent
-import android.graphics.Color
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import androidx.recyclerview.widget.RecyclerView
-import kotlinx.android.synthetic.main.view.view.*
-
-class Adapter(var data: List<CardInfo>) : RecyclerView.Adapter<Adapter.viewHolder>() {
-    class viewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
Class name should start with capital letter. Never use viewHolder as class name.Give some useful name